### ISSUE TITLE
Rule: Require block contents to be indented.

### ISSUE THREAD

**Issue by urbany** [rwjblue/ember-cli-template-lint#42]
Hi, not sure if this is on purpose but I'm getting no warnings when I test my build with this template:
```javascript
<div>
<div>
{{#if isSingleError}}
{{firstError}}
{{else if errors}}
{{errors}}
{{else}}
{{t 'no errors found'}}
{{/if}}
</div>
</div>
```
Should warn about multiple incorrect indentation correct?

I'm testing with ``ember serve`` and visiting: http://localhost:4200/tests

May be related to #36

**Comment by rwjblue** [comment]
Hehe, well, the current rule enforces that the closing tag/block is at the same indentation level as the opening tag/block (which technically is true here). 

I agree that this is unexpected, but it might be a different rule (enforce-nested-indentation maybe?)

**Comment by urbany** [comment]
Oh, ok that makes sense.
A new ``enforce-nested-indentation`` rule seems like a good idea, but might have to work with different types of indentation like: 2 spaces, 4 spaces, tab

**Comment by rwjblue** [comment]
Right, exactly

**Comment by rwjblue** [comment]
Updated the title to match other rule requests (and prevent myself from being confused as I keep reading the original title as a bug :crystal_ball:).

**Comment by rwjblue** [comment]
I ended up using the same rule (couldn't think of a good enough name for another one :stuck_out_tongue_winking_eye:).  Submitted https://github.com/rwjblue/ember-cli-template-lint/pull/51.
